### PR TITLE
Create staircase_climbs.py

### PR DESCRIPTION
**Pull request by satyabansahoo2000**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

 Function that returns the number of unique ways a person can climb the stairs with only 1 or 2 steps with a time complexity of O(n).

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)

### CODE REVIEW

**Comment by anushkrishnav** [comment]
Please add your file and link the program file to the README . Thank you

File: Python/other/staircase_climbs.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+(1, 1, 1, 1, 1), (2, 1, 1, 1), (1, 2, 1, 1), (1, 1, 2, 1), (1, 1, 1, 2), (2, 2, 1), (2, 1, 2), (1, 2, 2)
+'''
+# Time Complexity - O(n^2)
+# Space Complexion - O(1)
+def staircase(n: int) -> int:
+  if n <= 3:
+    return n
+  else:
+    return staircase(n-1)+staircase(n-2)
+
```
Comment (iamrajiv) [inline Python/other/staircase_climbs.py:18]: @hemraj4545 remove unnecessary lines

**Review by iamrajiv — CHANGES_REQUESTED** [review]

File: Python/other/staircase_climbs.py (hunk 1 above)
Comment (satyabansahoo2000) [reply Python/other/staircase_climbs.py:18]: removed else statement and idented the last return statement.

**Comment by satyabansahoo2000** [comment]
updated the readme.md file

**Pull request closed without merging**